Makefile

```diff
@@ -36,7 +36,7 @@ build/libprism.$(SOEXT): $(SHARED_OBJECTS)
 
 build/libprism.a: $(STATIC_OBJECTS)
 	$(ECHO) "building $@"
-	$(Q) $(AR) $(ARFLAGS) $@ $(STATIC_OBJECTS) $(Q1:0=>/dev/null)
+	$(Q) $(AR) $(ARFLAGS) $@ $(STATIC_OBJECTS)
```

Comment:
A quick text find shows that this was the only recipe where `Q1` is used, and the only other use of `Q1` is https://github.com/ruby/prism/blob/17762fa4c0b2c069c1a4283cb47bc5276d9dd0f5/Makefile#L6, which may now be flattenend. (I’m not aware if the `V`s and `Q`s are a convention.)

Reply:
For this matter, ba82ce9 introduces an acutal use for `V0`:
https://github.com/ruby/prism/blob/ba82ce961516d2495e94c496c8c0c8820344abcb/Makefile#L17